src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -355,7 +355,7 @@ TEST_F(ResumeCtrlTest, StartResumption_AppWithCommands) {
       .WillByDefault(ReturnRef(*mock_help_prompt_manager_));
 
   for (uint32_t i = 0; i < count_of_commands; ++i) {
-    EXPECT_CALL(*mock_app_, AddCommand(i, test_application_commands[i]));
+    EXPECT_CALL(*mock_app_, AddCommand(i + 1, test_application_commands[i]));
```

Comment:
@APipko  Maybe it's better to use `cmd_num` instead of `i` and `internal_id` instead of `i + 1`, as you do in `resume_ctrl_impl.cc` ?